sidebars.js

```diff
@@ -98,18 +97,18 @@ module.exports = {
             "components/modeler/web-modeler/milestones",
             "components/modeler/web-modeler/token-simulation",
             {
-              "Connectors": [
+              Connectors: [
                 "components/modeler/web-modeler/connectors/index",
                 "components/modeler/web-modeler/connectors/use-connectors",
                 {
                   "Available Connectors": [
                     "components/modeler/web-modeler/connectors/available-connectors/index",
                     "components/modeler/web-modeler/connectors/available-connectors/rest",
                     "components/modeler/web-modeler/connectors/available-connectors/sendgrid",
-                  ]
-                }
-              ]
-            }
+                  ],
+                },
+              ],
+            },
```

Comment:
❤️ yesssss, trailing commas

Reply:
👍 Of course, this is just the default setting. I think it's good, but if we prefer something else we can change it in the `.prettierrc.json`